clang/include/clang/Driver/Types.def

```diff
@@ -104,7 +104,9 @@ TYPE("sycl-header",              SYCL_Header,  INVALID,         "h",      phases
 TYPE("sycl-fatbin",              SYCL_FATBIN,  INVALID,         nullptr,  phases::Compile, phases::Backend, phases::Assemble, phases::Link)
 TYPE("tempfilelist",             Tempfilelist, INVALID,         "txt",    phases::Compile, phases::Backend, phases::Assemble, phases::Link)
 TYPE("tempentriesfilelist",      TempEntriesfilelist, INVALID,  "txt",    phases::Compile, phases::Backend, phases::Assemble, phases::Link)
+TYPE("tempAOCOfilelist",         TempAOCOfilelist, INVALID,     "txt",    phases::Compile, phases::Backend, phases::Assemble, phases::Link)
```

Comment:
I may well be missing something, but why can't `"tempfilelist"` be re-used instead?

Reply:
<code>"tempfilelist"</code> is already being used as an input type that is used for the <code>llvm-foreach</code> implementation.  Needed a new type to differentiate for the aoco files from the lib.

Reply:
Got it, thanks